tests/SprykerEcoTest/Zed/Heidelpay/Business/HeidelpayFacadeIsSalesOrderCaptureApprovedTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Heidelpay\Business;
+
+use Generated\Shared\Transfer\HeidelpayTransactionLogTransfer;
+use Generated\Shared\Transfer\PaymentTransfer;
+use Orm\Zed\Sales\Persistence\SpySalesOrder;
+use SprykerEco\Shared\Heidelpay\HeidelpayConfig;
+use SprykerEcoTest\Zed\Heidelpay\Business\DataProviders\PaymentBuilder;
+
+/**
+ * @group Functional
+ * @group SprykerEcoTest
+ * @group Zed
+ * @group Heidelpay
+ * @group Business
+ * @group HeidelpayFacadeIsSalesOrderCaptureApprovedTest
+ */
+class HeidelpayFacadeIsSalesOrderCaptureApprovedTest extends HeidelpayPaymentTest
```

Comment:
create `Facade` folder and move this class to it as  `IsSalesOrderCaptureApprovedTest` class respectively